src/github.com/matrix-org/dendrite/clientapi/routing/routing.go

```diff
@@ -406,4 +406,31 @@ func Setup(
 			}}
 		}),
 	).Methods(http.MethodGet, http.MethodOptions)
+
+	r0mux.Handle("/user/{userId}/rooms/{roomId}/tags",
+		common.MakeAuthAPI("account_3pid", authData, func(req *http.Request, device *authtypes.Device) util.JSONResponse {
```

Comment:
why is this `"account_3pid"`? Tags and 3PID's are quite different...

Reply:
I remember I was not sure about what spans were and missed that bit. Can I get any suggestions and background about it?